src/App.css

```diff
@@ -1,39 +1,48 @@
-.App {
-  text-align: center;
+.appContainer {
+  width: 100%;
+  height: 100vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: #fff;
+}
+
+.calculator {
+  /* padding: 20px; */
```

Comment:
- [ ] Kindly remove commented code from your codebase as it helps us have cleaner code.